src/lib/components/Loading/LoadingContainer.styles.ts

```diff
@@ -0,0 +1,18 @@
+import { makeStyles } from '@material-ui/core';
+
+export const useStyles = makeStyles(
+  () => ({
+    container: {
+      flexDirection: 'column',
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+    expandedContainer: {
+      flexDirection: 'column',
+      alignItems: 'center',
+      justifyContent: 'center',
+      flexGrow: 1,
+    },
+  }),
+  { name: 'LoadingContainer' },
+);
```

Comment:
if you don't need theming you can just pass config ojb
```suggestion
export const useStyles = makeStyles({
    container: {
      flexDirection: 'column',
      alignItems: 'center',
      justifyContent: 'center',
    },
    expandedContainer: {
      flexDirection: 'column',
      alignItems: 'center',
      justifyContent: 'center',
      flexGrow: 1,
    },
  },
  { name: 'LoadingContainer' },
);
```

Reply:
Done